docs/firebase-setup.md

```diff
@@ -44,6 +44,17 @@ Team Members
         - fileName [string]
         - rank [number]
 
+Dates
+- dates
+    - [document]
```

Comment:
Can we add more specific documentation here regarding what endTime, startTime, ... mean?

As they all have types of number, it's more useful to provide an example of what the data should look like.